xrviz/dashboard.py

```diff
@@ -97,16 +102,19 @@ def clear_series(self, *args):
 
     def link_aggregation_selectors(self, *args):
         """
-        To link aggregation selectors with cmap limits
```

Comment:
This is an internal method, could prepend `_` to the name and maybe remove docstring